src/styles/app/37-database-pickers.css

```
@@ -72,20 +83,65 @@
+		in srgb,
+		var(--interactive-accent) 24%,
+		var(--border-default)
+	);
+}
+
+.databaseSourceInlinePicker .databasePickerTriggerIcon {
+	width: 20px;
+	height: 20px;
+	border-radius: var(--radius-sm);
+	background: transparent;
+	color: currentColor;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)